cumulus_etl/batching.py

```diff
@@ -0,0 +1,118 @@
+"""Iteration utilities to support batching values."""
```

Comment:
This is mostly the same as the deleted `batching.py` file, git just figured they were so different it didn't map the move.

Changes:
- Renamed `batch_iterate` to `batch_iterate_streams` and expand its docstring
- Added a `batch_iterate` that returned an iterator of actual batches (instead of tuples of batches, which is what `batch_iterate_streams` does - for most normal use cases, you don't need streaming support - that's a complexity that only the ETL tasks need)
- Added `_force_aiter` which upgrades a boring old sync iterable into an async iterable. So now these batching methods can take either.